toy_2d/src/test_ilqr_2d.py

```diff
@@ -88,12 +107,12 @@
     # make he flag true 
     load_old_run = False  
     if(load_old_run):
-        file_name = "controls"
+        file_name = "controls" + str(test_number)
```

Comment:
If the name of the controls file changes per my recommendation above, then make sure this gets updated too so accessing old results doesn't break.